molgenis-ontology-core/src/main/java/org/molgenis/ontology/beans/ComparableEntity.java

```diff
@@ -3,38 +3,52 @@
 import java.math.BigDecimal;
 
 import org.molgenis.data.Entity;
+import org.molgenis.data.support.MapEntity;
+import org.molgenis.ontology.repository.OntologyTermQueryRepository;
 
-public class ComparableEntity implements Comparable<ComparableEntity>
+public class ComparableEntity extends MapEntity implements Comparable<ComparableEntity>
```

Comment:
Wouldn't a Comparator<Entity> be more obvious and less intrusive than creating a whole new entity class?